<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ROUTE_Editor_ui</class>
 <widget class="QDialog" name="ROUTE_Editor_ui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>574</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>8000</width>
    <height>6000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Route Parameters</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>551</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Route</string>
    </attribute>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>581</width>
       <height>511</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>471</width>
        <height>71</height>
       </rect>
      </property>
      <property name="title">
       <string>Nom</string>
      </property>
      <widget class="QLineEdit" name="editName">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>311</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="editBoat">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>20</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>360</y>
        <width>561</width>
        <height>151</height>
       </rect>
      </property>
      <property name="title">
       <string>Options</string>
      </property>
      <widget class="QCheckBox" name="editFrozen">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>111</width>
         <height>18</height>
        </rect>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string>Figer la route</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="editCoasts">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>20</y>
         <width>411</width>
         <height>18</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Les cotes invibles a l'ecran ne sont pas prises en compte</string>
       </property>
       <property name="text">
        <string>Detecter les collisions avec les cotes visibles a l'ecran</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QCheckBox" name="hidePois">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>221</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Cacher les POIs intermediaires</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="vacStep">
       <property name="geometry">
        <rect>
         <x>500</x>
         <y>50</y>
         <width>42</width>
         <height>22</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>270</x>
         <y>50</y>
         <width>221</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Nbre de vac par pas de calcul</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="hidden">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>221</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Cacher la route et tous ses POIs</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="useVbvmgVlm">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>80</y>
         <width>271</width>
         <height>17</height>
        </rect>
       </property>
       <property name="palette">
        <palette>
         <active>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </active>
         <inactive>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>0</red>
             <green>0</green>
             <blue>255</blue>
            </color>
           </brush>
          </colorrole>
         </inactive>
         <disabled>
          <colorrole role="WindowText">
           <brush brushstyle="SolidPattern">
            <color alpha="255">
             <red>120</red>
             <green>120</green>
             <blue>120</blue>
            </color>
           </brush>
          </colorrole>
         </disabled>
        </palette>
       </property>
       <property name="toolTip">
        <string>Ce bouton a 3 etats, selon que vous utilisez la formule qtVlm, une formule VLM optimisee, ou la formule VLM originale. La formule VLM optimisee donne en principe exactement les memes resultats que la formule non optimisee.</string>
       </property>
       <property name="statusTip">
        <string/>
       </property>
       <property name="text">
        <string>Formule VLM pour le VB-VMG</string>
       </property>
       <property name="tristate">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="labelTackGybe">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Empannages/Virements</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="speedLossOnTack">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>110</y>
         <width>61</width>
         <height>21</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Pourcentage de la polaire a appliquer pendant une vacation</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
      <widget class="QCheckBox" name="autoRemove">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>100</y>
         <width>261</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Retirer les POIs avant le WP-VLM</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="autoAt">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>120</y>
         <width>271</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Renseigner le cap a suivre (@) des POIs</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_8">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>250</y>
        <width>561</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Taille et couleur</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>541</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_9">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>140</y>
        <width>391</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Date et heure de depart de la route</string>
      </property>
      <widget class="QWidget" name="formLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>371</width>
         <height>78</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="1" column="0">
         <widget class="QRadioButton" name="editGrib">
          <property name="text">
           <string>Depuis la date du grib</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QRadioButton" name="editDate">
          <property name="text">
           <string>Date fixe</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QDateTimeEdit" name="editDateBox">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="displayFormat">
           <string>M/d/yyyy hh:mm:ss</string>
          </property>
          <property name="timeSpec">
           <enum>Qt::UTC</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QRadioButton" name="editVac">
          <property name="text">
           <string>A partir de la prochaine vacation</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>171</width>
        <height>101</height>
       </rect>
      </property>
      <property name="title">
       <string>Point de depart de la route</string>
      </property>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>7</x>
         <y>30</y>
         <width>161</width>
         <height>49</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QRadioButton" name="startFromBoat">
          <property name="text">
           <string>Du bateau</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="startFromMark">
          <property name="text">
           <string>Depuis la 1ere marque</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QPushButton" name="btOk">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>8</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="Simplifier">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>40</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Simplifie la route en supprimant les POIs qui ne changent pas l'eta finale.</string>
      </property>
      <property name="text">
       <string>Simplifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btCancel">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="pilototo">
      <property name="geometry">
       <rect>
        <x>490</x>
        <y>70</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>170</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>170</red>
            <green>0</green>
            <blue>0</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>120</red>
            <green>120</green>
            <blue>120</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string>Pilototo</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Pilototo</string>
    </attribute>
    <widget class="QPushButton" name="Envoyer">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>420</y>
       <width>251</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Envoyer les ordres</string>
     </property>
    </widget>
    <widget class="QPushButton" name="defaultOrders">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>20</y>
       <width>251</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Charger automatiquement</string>
     </property>
    </widget>
    <widget class="QTreeView" name="pilotView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>591</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="customOrders">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>50</y>
       <width>251</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Charger les POIs preselectionnes</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>btDeleteClicked()</slot>
  <slot>GybeTack(int)</slot>
 </slots>
</ui>
